app/jobs/send_welcome_email.rb

```diff
@@ -1,5 +1,6 @@
 class SendWelcomeEmail < Struct.new(:person_id, :community_id)
 
+  include SessionContextSerializer
```

Comment:
Yeah, that's what I figured. I think I would have left it away by default simply to be explicit about where it is required (and have client middleware log a warning / raise when there's no auth context). The small risk with this approach is that it gets included without understanding what it is or does by those who come later. Either way works of course and don't feel particularly strongly about this.